src/SearchBox.css

```diff
@@ -0,0 +1,30 @@
+.searchBox__container{
+    height: 90px;
+    background-color:#F8F8F8;
+    display:flex;
+    align-items: center;
+}
+.search__items{
+    display:flex;
+    align-items: center;
+    flex:1;
+    margin-right: 7px;
+    margin-left: 7px;
+    background-color: white;
+    border-radius: 20px;
+    height:30px;
+}
+.search__items > input{
+    flex:5;
+    border: none;
+    border-radius: 20px;
+    outline-width: 0;
+}
+
+.search__items > svg{
+    height: 100%;
+    flex:1;
+    background-color:white;
+    border-radius: 20px;
```

Comment:
NO need of height, flex, background-color, border-radius

Reply:
then how to apply styling on search icon? Space, box around it etc.

Reply:
there is no need of styling here, parent style will do the necessary